### PR TITLE
Fix missing link by redirecting to a TODO page

### PR DESCRIPTION
**Pull request by mathbagu**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **What is the current behavior?** (You can also link to an open issue here)
Some pages were orphans (currentLimits), people were unable to read them. There are also non set links. For a reader it's difficult to understand that the requested page doesn't exist.

* **What is the new behavior (if this is a feature change)?**
The currentLimits page is not referenced by Line, two- and three-windings transformers.
All the links redirect the reader to a page, and a _TODO_ page for non existing page.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Other information**:

### CODE REVIEW

File: docs/configuration/modules/remote-service.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ title: remote-service
 layout: default
 ---
 
-The `remote-service` module is used by [AFS]() to define a remote AFS server.
+The `remote-service` module is used by [AFS](../../todo.md) to define a remote AFS server.
```
Comment (MioRtia) [inline docs/configuration/modules/remote-service.md]: this page for AFS exists, redirect to it instead:
`../../afs/index.md`

File: docs/configuration/modules/remote-service.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ title: remote-service
...
 
-Read the [documentation]() page to learn more about the remote AFS server implementation.
+Read the [documentation](../../todo.md) page to learn more about the remote AFS server implementation.
```
Comment (MioRtia) [inline docs/configuration/modules/remote-service.md]: this page for remote AFS exists, redirect to it instead:
`../../afs/afs-remote.md`

File: docs/tools/afs.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ todo:
     - add links to missing pages
 ---
 
-The `afs` command performs some basic actions on a [AFS]() drive:
+The `afs` command performs some basic actions on a [AFS](../afs/index.md) drive:
 - list the content of an AFS folder
 - archive/restore an AFS content from/to a directory for backup purposes
 
-Read this [tutorial]() to learn how to configure an AFS drive. 
+Read this [tutorial](../todo.md) to learn how to configure an AFS drive.
```
Comment (MioRtia) [inline docs/tools/afs.md]: this page exists, redirect to it instead:
`../tutorials/afs/afs.md`

File: docs/configuration/modules/remote-service.md (hunk 1 above)
Comment (mathbagu) [reply docs/configuration/modules/remote-service.md]: Change to `../../afs/afs-remote.md` seems better. What do you think?

**Pull request merged**